Documentation/Blazorise.Docs/Components/Commercial/PlanSwitch.razor.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+using Blazorise.Docs.Services;
+using Microsoft.AspNetCore.Components;
+
+namespace Blazorise.Docs.Components.Commercial;
+
+public partial class PlanSwitch : IDisposable
+{
+    private Background GetSelectedBackground( string value ) => SelectedValue == value ? Background.Primary : Background.Default;
+
+    private TextColor GetTextSelectedColor( string value ) => SelectedValue == value
```

Comment:
Same thing, don't understand why SelectedValue == value turns into TextColor.White?

Reply:
White looks better on the primary purple color. That's why it is white in selected mode.